frontend/public/components/utils/request-size-input.tsx

```diff
@@ -20,7 +20,9 @@ export const RequestSizeInput: React.FC<RequestSizeInputProps> = ({
   testID,
 }) => {
   const [unit, setUnit] = React.useState(defaultRequestSizeUnit);
-  const [value, setValue] = React.useState(parseInt(defaultRequestSizeValue, 10));
+  const [value, setValue] = React.useState(
+    defaultRequestSizeValue && parseInt(defaultRequestSizeValue, 10),
```

Comment:
Actually even that isn't perfect since we want to handle parse errors if I pass a string like `"foo"`. It would be better to parse it before calling `React.useState` and check `Number.isFinite`.

Reply:
Updated.